src/Swarm/TUI/Controller.hs

```diff
@@ -261,7 +261,8 @@ handleMainEvent ev = do
     MetaKey 'h' -> do
       t <- liftIO $ getTime Monotonic
       uiState . uiHideRobotsUntil .= t + TimeSpec 2 0 -- hide for two seconds
-      -- pausing and stepping
+      invalidateCacheEntry WorldCache
```

Comment:
@xsebek I wondered why it took up to a second or so for robots to disappear after I hit M-h.  Then I figured it out --- we try hard NOT to redraw the world on every event because it is slow.  So when the world actually needs to be redrawn we have to signal it explicitly.  I fixed it by calling `invalidateCacheEntry WorldCache` here.

I also realized we should do the same fix for Ctrl-v.  Don't know whether you want to just include that fix with this PR or put it in a separate one.

Reply:
Interesting, I originally had code to not change anything for one second.

Might as well include it again if some player decides to hold down the <kbd>H</kbd> key. 😅